src/drivers/bitbucket_cloud.js

```diff
@@ -21,13 +21,35 @@ class BitBucketCloud {
     const { project_path } = this;
     const { commit_sha, report } = opts;
 
-    // Print some variables
-    const endpoint = `/repositories/${project_path}/commit/${commit_sha}/comments/`;
-    const body = JSON.stringify({ content: { raw: report } });
-
-    const output = await this.request({ endpoint, method: 'POST', body });
-
-    return output;
+    // Make a comment in the commit
+    const commit_endpoint = `/repositories/${project_path}/commit/${commit_sha}/comments/`;
+    const commit_body = JSON.stringify({ content: { raw: report } });
+    const commit_output = await this.request({
+      endpoint: commit_endpoint,
+      method: 'POST',
+      body: commit_body
+    });
+
+    // Check for a corresponding PR. If it exists, also put the comment there.
+    const get_pr_endpt = `/repositories/${project_path}/commit/${commit_sha}/pullrequests`;
+    const pr_out = await this.request({ endpoint: get_pr_endpt });
+    if (pr_out.values) {
+      // Get PR ID
+      const pr_id = pr_out.values[0].id;
+      // Append a watermark to the report with a link to the commit
+      const commit_link = commit_sha.substr(0, 7);
+      const long_report = `${commit_link}\n  ${report}`;
```

Comment:
It's not properly working. Its not generating a new line and the whole report looks broken we have to find the way to create a new line. 

![image](https://user-images.githubusercontent.com/414967/99702341-eecf7880-2a95-11eb-860b-2026c112ffa5.png)